torch/csrc/distributed/c10d/comm.cpp

```diff
@@ -61,9 +61,12 @@ void broadcast_coalesced(
   // Coalesce tensors into buckets taking into account the maximum buffer size.
   // This routine is multi-device aware, so the tensors can be split across
   // multiple devices and can contain a mix of CPU and CUDA tensors.
-  const auto buckets =
+  const auto tup =
```

Comment:
shall we make this var name more readable? (e.g., `buckets_and_size_limits`) Or we can directly run `std::tie` here?